toplevel/dune

```diff
@@ -3,9 +3,8 @@
  (public_name coq.toplevel)
  (synopsis "Coq's Interactive Shell [terminal-based]")
  (wrapped false)
- ; num still here due to some plugins using it
- (libraries num coq.stm))
-; Interp provides the `zarith` library to plugins, we could also use
-; -linkall in the plugins file, to be discussed.
+ (libraries coq.stm))
+; Interp does provides the `zarith` library to plugins, we could also
```

Comment:
Grammatical typo: "does provides" -> "does provide". But IMHO, just "provides" was simple enough. It doesn't bring anything to add "does".

Reply:
Thanks.